lib/aur-repo

```diff
@@ -186,14 +186,16 @@ if [[ ! $db_name ]]; then
 
 # check $db_name is a configured pacman repository (local or remote, #1113)
 elif ! [[ ${conf_sync_serv[$db_name]} ]]; then
-    printf >&2 "%s: repository %q not configured\n" "$err_prefix" "$db_name"
-    exit 2
+    printf >&2 "warning: repository %q not configured\n" "$db_name"
```

Comment:
May want to take the opportunity here to replace `%q` with `%s`.

Reply:
Made the change. There's some other places left with `%q` that might be worth a look later.